tests/framework/checks/skips.go

```diff
@@ -225,3 +227,9 @@ func SkipIfARM64(arch string, message string) {
 		ginkgo.Skip("Skip test on arm64: " + message)
 	}
 }
+
+func SkipIfInPlaceHotplugNICs() {
+	if strings.ToLower(os.Getenv("TEST_IN_PLACE_HOTPLUG_NICS")) != "true" {
```

Comment:
maybe use [os.LookupEnv](https://pkg.go.dev/os#LookupEnv) instead ? i.e. if it is defined, run them, otherwise skip ? Really **not** a must.

Plus, I really dislike the name you chose :) 

Maybe `DYNAMIC_NICS_SUPPORT` , `TEST_NIC_HOTPLUG` , ... (the part that aggravates me is the `in place` thing).

Reply:
In this case, it's easier to work with `os.Getenv` because it's not necessary to distinguish between an empty value and an unset value.

I removed the in-place naming for now.